packages/sdk/odahuflow/sdk/clients/api_aggregated.py

```diff
@@ -269,7 +269,7 @@ def parse_resources_file_with_one_item(path: str) -> OdahuflowCloudResourceUpdat
     """
     resources = parse_resources_file(path)
     if len(resources.changes) != 1:
-        raise Exception('{!r} should contain 1 item, but {!r} found'.format(path, len(resources)))
+        raise Exception(f'{path!r} should contain 1 item, but {len(resources)} found')
```

Comment:
```suggestion
        raise Exception(f'{path!r} should contain 1 item, but {len(resources)!r} found')
```
Or it doesn't make a change?

Reply:
len returns an int, so yeah, there is no point in repr